getting-started.md

```diff
@@ -0,0 +1,61 @@
+
+- Getting Started: A Novella
+Last updated 10/7/19
+
+- Repo Setup:
+  
+  - Note: NOT recommend for devs to use React Native Debugger. It will be too slow. We have our own system of logging in the console that works great!
+  - Note: The repo on GitHub is called phd-mobile.
+  
+  1. Create a GitHub account with your yum email. This is the email that will get the invite to our internal repos. Teams message     Stephen Miller with your new GitHub account info and he will add you.
```

Comment:
Maybe it's just on my screen but there's this long gap between `Teams message` and `Stephen`